### PR TITLE
Adding info and ssh commands to automate-cli

### PR DESCRIPTION
**Pull request by FaizanSRE**
Signed-off-by: FaizanSRE <ffulara@msystechnologies.com>

### :nut_and_bolt: Description: What code changed, and why?
This will map the a2ha command "automate-cluster-ctl info " and "automate-cluster-ctl ssh "
 commands to "chef-automate info " and "chef-automate ssh" respectively.
info command is used to get details of all the components involved in automate-HA
ssh command is used to login to servers of HA components like elasticsearch , postgressql,chef_server, automate

### :athletic_shoe: How to Build and Test the Change
 1. hab studio enter
 2. start_all_services
 3. build components/automate-cli
Then now you can check the above commands 

1. chef-automate ssh postgresql
desired o/p if deployment is completed - will ssh into postgresql server
 desired o/p if deployment is not-completed - 
```
 [2][default:/src:0]# chef-automate ssh postgresql
Login into cluster components 
[Tue Sep 14 13:15:56 2021] › ⚠ warning Secret storage has not been configured yet, please run `automate-cluster-ctl secrets init`
ERROR: No servers where found for postgresql, please ensure that a deploy has completed successfully.
```
5. chef-automate info
Desired o/p --Details of all the components server info if package is deployed
Will give empty o/p when deployment is not done

### :white_check_mark: Checklist

**All PRs** must tick these:

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [x] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [x] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [x] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [x] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [x] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [x] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [ ] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### CODE REVIEW

**Comment by FaizanSRE** [comment]
@vivek-yadav @kalroy I have added the help information for info command , as discussed yesterday. So now we don't have anything pending, so please have a look and approve it.

File: components/automate-cli/cmd/chef-automate/Ssh-ha-template.go [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    chef-automate [OPTIONS] NAME [COMMAND]
+
+Parameters:
+    NAME                             Name of the server type to access, for example automate, chef_server, elasticsearch or postgresql
+    [COMMAND]                        Command to run on the remote server
+
+Options:
+    -o, --ssh-options OPTIONS        Additional options to pass to ssh
+    -l, --list                       List available servers for ssh
+    -h, --help                       print help
```
Comment (kalroy) [inline components/automate-cli/cmd/chef-automate/Ssh-ha-template.go:14]:
```suggestion
    -h, --help                       Print help
```

File: components/automate-cli/cmd/chef-automate/Ssh-ha.go [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+)
+
+func init() {
+	SshCmd.SetUsageTemplate(SshCommandHelpDocs)
+	RootCmd.AddCommand(SshCmd)
+}
+
+var SshCmd = &cobra.Command{
+	Use:   "ssh",
+	Short: "set Ssh env for automate HA.",
```
Comment (kalroy) [inline components/automate-cli/cmd/chef-automate/Ssh-ha.go:21]:
```suggestion
	Short: "Set Ssh env for automate HA.",
```

File: components/automate-cli/cmd/chef-automate/Ssh-ha.go [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+	Long:  "set up automate ha clusterSsh",
```
Comment (kalroy) [inline components/automate-cli/cmd/chef-automate/Ssh-ha.go:22]:
```suggestion
	Long:  "Set up automate ha clusterSsh",
```

File: components/automate-cli/cmd/chef-automate/info-ha.go [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+	RunE: runInfoConfigCmd,
+}
+
+func runInfoConfigCmd(cmd *cobra.Command, args []string) error {
+	return executeInfoCommand()
+}
+
+func executeInfoCommand() error {
+	writer.Printf("Automate HA info \n")
+	//args = append([]string{"info"})
```
Comment (kalroy) [inline components/automate-cli/cmd/chef-automate/info-ha.go:36]: Remove commented codes

File: components/automate-cli/cmd/chef-automate/info-ha.go [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+	var stderr bytes.Buffer
+	c.Stdout = &out
+	c.Stderr = &stderr
+	err := c.Run()
+	if err != nil {
+		writer.Printf(stderr.String())
+		return status.Wrap(err, status.CommandExecutionError, "please refer \n"+infoHelpDocs)
+	}
+	writer.Print(out.String())
+	//writer.Printf("A2HA new secret set. %d, exiting\n", c.Process.Pid)
```
Comment (kalroy) [inline components/automate-cli/cmd/chef-automate/info-ha.go:50]: Remove commented codes

File: components/automate-cli/cmd/chef-automate/info-ha.go [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+var infoCmd = &cobra.Command{
+	Use:   "info",
+	Short: "InFo about automate HA",
+	Long:  "Info for automate HA cluster",
+	Annotations: map[string]string{
+		NoCheckVersionAnnotation: NoCheckVersionAnnotation,
+	},
+	//Args: cobra.RangeArgs(0, 2),
```
Comment (kalroy) [inline components/automate-cli/cmd/chef-automate/info-ha.go:26]: Remove

**Review by kalroy — CHANGES_REQUESTED** [review]

File: components/automate-cli/cmd/chef-automate/info-ha.go (hunk 6 above)
Comment (FaizanSRE) [reply components/automate-cli/cmd/chef-automate/info-ha.go]: Done

File: components/automate-cli/cmd/chef-automate/info-ha.go (hunk 5 above)
Comment (FaizanSRE) [reply components/automate-cli/cmd/chef-automate/info-ha.go]: Done

File: components/automate-cli/cmd/chef-automate/info-ha.go (hunk 6 above)
Comment (FaizanSRE) [reply components/automate-cli/cmd/chef-automate/info-ha.go]: @kalroy Commited all suggested changes, please have a look

**Review by kalroy — APPROVED** [review]

**Review by punitmundra — APPROVED** [review]
rebase with the latest code and make sure that buildkite should passed

**Pull request merged**